src/Stack/Build/Execute.hs

```diff
@@ -1974,7 +1974,7 @@ extraBuildOptions :: (HasEnvConfig env, HasRunner env)
                   => WhichCompiler -> BuildOpts -> RIO env [String]
 extraBuildOptions wc bopts = do
     colorOpt <- appropriateGhcColorFlag
-    let ddumpOpts = " -ddump-hi -ddump-to-file"
+    let ddumpOpts = " -keep-hi-files"
```

Comment:
This is the default behavior of GHC according to the manual. I don't mind being explicit, but could you add a comment explaining the reason for reinforcing the default?

Reply:
I did not know that! Will change it!